dags/oss_know/oss_know_dags/dags_zulip/dag_crawl_stream.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+import time
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from opensearchpy import OpenSearch, helpers
+
+from oss_know.libs.base_dict.variable_key import ZULIP_API_KEYS, NEED_INIT_ZULIP, OPENSEARCH_CONN_DATA
+
+with DAG(
+    dag_id = 'zulip_crawl_stream',
+    schedule_interval=None,
+    start_date=datetime(2023, 3, 1),
+    catchup=False,
+    tags=['zulip']
+) as dag:
+    def init_zulip_crawl_stream(ds, **kwargs):
+        return 'Start init_zulip_crawl_stream'
+    
+    op_init_zulip_crawl_stream = PythonOperator(
+        task_id='init_zulip_crawl_stream',
+        python_callable=init_zulip_crawl_stream
+    )
+
+    def do_zulip_crawl_stream(params):
+        from airflow.models import Variable
+        from oss_know.libs.zulip import crawl_stream
+        from oss_know.libs.util.zulip import get_api
+        owner = params["owner"]
+        repo = params["repo"]
+        site = params["site"]
+        opensearch_conn_info = Variable.get(OPENSEARCH_CONN_DATA, deserialize_json=True)
+        api_key = Variable.get(ZULIP_API_KEYS, deserialize_json=True)
+
+        client = get_api(email=api_key["email"], api_key=api_key["api_key"], site=site)
+        crawl_zulip_stream_info = crawl_stream.crawl_zulip_stream(client=client,
```

Comment:
Since function `crawl_zulip_stream` always return `None`, so just call it without accessing return value.